Build fails on s390x:

Build fails on s390x:
```
error[E0425]: cannot find value `EFIPN` in this scope
   --> /builddir/build/BUILD/rpm-ostree-2023.5.38.g0cbcf4ae/vendor/bootc-lib/src/install/baseline.rs:247:13
    |
247 |             EFIPN,
    |             ^^^^^ not found in this scope
error[E0425]: cannot find value `EFIPN_SIZE_MB` in this scope
   --> /builddir/build/BUILD/rpm-ostree-2023.5.38.g0cbcf4ae/vendor/bootc-lib/src/install/baseline.rs:248:26
    |
34  | pub(crate) const BOOTPN_SIZE_MB: u32 = 510;
    | ------------------------------------------- similarly named constant `BOOTPN_SIZE_MB` defined here
...
248 |             format!("0:+{EFIPN_SIZE_MB}M"),
    |                          ^^^^^^^^^^^^^ help: a constant with a similar name exists: `BOOTPN_SIZE_MB`
error[E0425]: cannot find value `EFIPN` in this scope
   --> /builddir/build/BUILD/rpm-ostree-2023.5.38.g0cbcf4ae/vendor/bootc-lib/src/install/baseline.rs:252:32
    |
252 |         Some(format!("{device}{EFIPN}"))
    |                                ^^^^^ not found in this scope
For more information about this error, try `rustc --explain E0425`.
error: could not compile `bootc-lib` due to 3 previous errors
Caused by:
  process didn't exit successfully: `rustc --crate-name bootc_lib --edition=2021 /builddir/build/BUILD/rpm-ostree-2023.5.38.g0cbcf4ae/
```
This blocks an rpm-ostree release as we are vendoring bootc in rpm-ostree.

--- a/lib/src/install/baseline.rs
+++ b/lib/src/install/baseline.rs
@@ -33,16 +33,8 @@ pub(crate) const BOOTPN: u32 = 3;
 // This ensures we end up under 512 to be small-sized.
 pub(crate) const BOOTPN_SIZE_MB: u32 = 510;
 pub(crate) const ROOTPN: u32 = 4;
-#[cfg(any(target_arch = "x86_64", target_arch = "aarch64"))]
 pub(crate) const EFIPN: u32 = 2;
-#[cfg(any(target_arch = "x86_64", target_arch = "aarch64"))]
 pub(crate) const EFIPN_SIZE_MB: u32 = 512;
-#[cfg(target_arch = "aarch64")]
-pub(crate) const RESERVEDPN: u32 = 1;
-#[cfg(target_arch = "ppc64")]
-pub(crate) const PREPPN: u32 = 1;
-#[cfg(target_arch = "ppc64")]
-pub(crate) const RESERVEDPN: u32 = 1;
 
 #[derive(clap::ValueEnum, Debug, Copy, Clone, PartialEq, Eq, Serialize, Deserialize)]
 #[serde(rename_all = "kebab-case")]

